src/main/java/no/nav/fo/veilarbregistrering/oppgave/OppgaveRouter.java

```diff
@@ -82,8 +83,9 @@ public OppgaveRouter(
             Optional<Enhetsnr> enhetsnr = hentEnhetsnummerForSisteArbeidsforholdTil(bruker);
             if (enhetsnr.isPresent()) {
                 reportTags(OPPGAVE_ROUTING_EVENT, Enhetsnummer_Funnet);
+                return enhetsnr;
             }
-            return enhetsnr;
+            return of(Enhetsnr.of("2930")); // Intern Brukerstøtte
```

Comment:
I stedet for å ha intern brukerstøtte som en kommentar her, foreslår jeg å lage en static method på Enhet som bruker navnet internBrukerstøtte, og som oversetter til 2930 på innsiden. `Enhetsnr.internBrukerstotte()`

Reply:
Meget god idé! 👌